dashboards/node.libsonnet

```diff
@@ -129,12 +129,11 @@ local gauge = promgrafonnet.gauge;
       local diskSpaceUsage = gauge.new(
         'Disk Space Usage',
         |||
-          (
-            sum(node_filesystem_size{%(nodeExporterSelector)s, device!="rootfs", instance="$instance"})
-          - sum(node_filesystem_avail{%(nodeExporterSelector)s, device!="rootfs", instance="$instance"})
-          ) * 100
+          (1 - max (
```

Comment:
I'm not sure this query is correct. `max(avail / size)` gives us the devices that have the most available disk, but I think the heuristic you meant to use was using the biggest filesystem by disk, but even then what about if you have one device but two equally large filesystems?

Reply:
Agreed, let me revisit this one actually. It "worked on my machine" but it might that I missed something. The `max` is as below to remove duplicates caused by bind-mounts of node exporter in a container.

Reply:
/cc @tomwilkie @brancz so, I think this query doesn't really make much sense right now. I suggest we think about removing the "Nodes" dashboard at all and rather merge all useful things into the USE dashboards instead.

For the time being I will at least reference the supported filesystem types config to retain the current semantics.

Reply:
Yeah I think the node dashboard doesn't show anything the use dashboard doesn't so I would vote to remove as well. In a follow up though.